internal/acceptance_test/resource_host_test.go

```diff
@@ -156,6 +187,45 @@ func testWaitUntilHostReady(rsrc string) resource.TestCheckFunc {
 	}
 }
 
+// testVerifyHostReady checks if the host was created successfully and
+// is in the 'Ready' state.
+func testVerifyHostReady(rsrc string) resource.TestCheckFunc {
```

Comment:
rsrc is a strange name for host. Is that some TF standard? Oh, resource? Maybe just "hostname"?

Reply:
existing test...will rename the attribute